Description: Increase the limit of responses
 The sushi code is written to support receiving multiple matches for its query
 to MB: it loops through and picks the first match which has an ASIN.
 .
 However, when the actual query is made, the limit on responses is set to one,
 so only one result is ever returned.
 .
 In the case of many albums, there are multiple matches, sometimes with bootlegs
 or just with records which aren't fleshed out with an ASIN.
Author: Jon Dowland <jmtd@debian.org>
Forwarded: No
Index: sushi-0.4.1/src/libsushi/sushi-cover-art.c
===================================================================
--- sushi-0.4.1.orig/src/libsushi/sushi-cover-art.c	2012-06-19 21:43:43.749178148 +0200
+++ sushi-0.4.1/src/libsushi/sushi-cover-art.c	2012-06-19 21:44:54.721874708 +0200
@@ -229,7 +229,7 @@
   param_values[0] = g_strdup_printf ("artist:\"%s\" AND release:\"%s\"", job->artist, job->album);
 
   param_names[1] = g_strdup ("limit");
-  param_values[1] = g_strdup ("1");
+  param_values[1] = g_strdup ("10");
 
   param_names[2] = NULL;
   param_values[2] = NULL;
